### PR TITLE
Added "clear buttons. Buttons display change on filter set. Major CSS formatting fix

### PR DESCRIPTION
**Pull request by rtsuzaki**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React from 'React';
```
Comment (clfhhc) [inline client/src/components/App.jsx]: why is it capital letter R in module?

File: client/src/components/BedroomsCounter.jsx [hunk 2]
```diff
@@ -1,20 +1,35 @@
 import React from 'React';
 
 const BedroomsCounter = (props) => {
+  if (props.bedMin === 1) {
```
Comment (clfhhc) [inline client/src/components/BedroomsCounter.jsx]: why not use state to keep track of this? or use turnery directly in your jsx

File: client/src/components/RoomListCarousel.jsx [hunk 3]
```diff
@@ -7,25 +7,28 @@ const RoomListCarousel = (props) => {
   let firstHalf = props.displayedRooms.slice(0,halfLength);
   let secondHalf = props.displayedRooms.slice(halfLength,props.displayedRooms.length);
   return (
-    <div className='carousel'>
+    <div className="carousel-content">
       <RoomListCarouselArrow
           direction="left"
           clickFunction={props.previousSlide}
-          glyph="&#9664;" />
+          arrow={<path d="m 13.7 16.29 a 1 1 0 1 1 -1.42 1.41 l -8 -8 a 1 1 0 0 1 0 -1.41 l 8 -8 a 1 1 0 1 1 1.42 1.41 l -7.29 7.29 Z"></path>}
```
Comment (clfhhc) [inline client/src/components/RoomListCarousel.jsx]: This is awesome. Is this some secrete code to modify the span element?

File: client/src/components/BedroomsCounter.jsx (hunk 2 above)
Comment (rtsuzaki) [reply client/src/components/BedroomsCounter.jsx]: great idea thanks. I will make that change.

**Pull request merged**